students/vvinodh/Lesson2/fibo_series.py

```diff
@@ -0,0 +1,14 @@
+def fibo(n):
+    #initial assignment based on where the series need to start
+    a = 0
+    b = 1
+    # If the required index is 0th position print the first fibonacci number
+    if n == 0:
+        print (a)
+    else:
+    # Loop to move the latest number to prior and new number to the next calculated variable
+        for i in range(n):
+                next = a + b
+                a = b
+                b = next
```

Comment:
notice how github highlights `next` in blue? this means that it is a built-in, by using that as a variable you are overwriting built-in function; this should be avoided by using a different var name.